Simplenote/src/main/java/com/automattic/simplenote/utils/DisplayUtils.java

```diff
@@ -72,8 +75,15 @@ public static int getChecklistIconSize(Context context) {
         if (context == null) {
             return 18;
         }
-
         return DisplayUtils.dpToPx(context, PrefUtils.getFontSize(context)) + DisplayUtils.dpToPx(context, 6);
     }
-}
 
+  // Disable screenshots if app is lock is on
```

Comment:
```suggestion
  // Disable screenshots if app PIN lock is on
```

I think this comment has a typographical error.  Should it be "// Disable screenshots if app PIN lock is on" instead?

Reply:
Yes you are right. Sorry it slipped my mind.